tests/sv.ts

```diff
@@ -0,0 +1,14 @@
+import { runInRepo } from '../utils.ts'
+import { RunOptions } from '../types.js'
+
+export async function test(options: RunOptions) {
+	await runInRepo({
+		...options,
+		repo: 'sveltejs/cli',
+		branch: 'main',
+		build: 'pnpm build',
+		beforeTest:
+			'pnpm playwright install chromium && pnpx storybook@latest --version', // prefetch the storybook cli to reduce fetching errors in tests
```

Comment:
is the storybook cli used by the tests also always "latest"? what happens if they release a new major?

If possible make it a devDependency and call it via `pnpm storybook` just like playwright

Reply:
I do see the point you are making. Our `storybook` add-on just executes their cli, and does nothing else. That's why we decided to leave it up to the storybook team (and Jeppe is super responsive)

That said, adding it to the devDeps of the cli won't make the situation much better, as we would still need to remember to update the version there. But it would already be simple, since it's only in one repo.

But I have another idea, will check and report back.